File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/service/invocation/exception/ParamException.java
package org.javamaster.invocationlab.admin.service.invocation.exception;

import org.javamaster.invocationlab.admin.service.invocation.ResponseCode;
import lombok.Getter;

/**
 * @author yudong
 * 参数解析异常
 */
@Getter
public class ParamException extends Exception {

    private final int code;

    public ParamException(String msg) {
        super(msg);
        this.code = ResponseCode.SYSTEM_ERROR.getCode();
    }

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/service/invocation/exception/ParamException.kt
package org.javamaster.invocationlab.admin.service.invocation.exception

import org.javamaster.invocationlab.admin.service.invocation.ResponseCode

/**
 * @author yudong
 * 参数解析异常
 */
class ParamException(msg: String?, val code: Int = ResponseCode.SYSTEM_ERROR.code) : Exception(msg)
